binarytree/__init__.py

```diff
@@ -457,14 +458,15 @@ def inspect(bt):
     min_leaf_depth = 0
     current_depth = -1
     current_nodes = [bt]
-
+    
     while current_nodes:
 
         null_encountered = False
         current_depth += 1
         next_nodes = []
 
         for node in current_nodes:
+            number_childs = 0
```

Comment:
Could you change the variable name here to, say, `num_of_children` instead? Just to be grammatically correct. Thanks!